README.md

```diff
@@ -2,11 +2,10 @@
 
 This is a ruby gem that basically wraps the msfrpc-client gem. 
 
-It was primarily created to automate workspace creation and automatic import of nexpose scan data from use in 
+It was primarily created to automate workspace creation and automatic import of Nexpose scan data from use in 
 a Jenkins automated CI/CD environment. 
 
-Basically this allows you to attach metasploit to your Continuous Delivery/Continuous integration pipeline. 
-Though it can be used for other purposes.
+Basically this allows you to attach Metasploit to your Continuous Delivery/Continuous integration pipeline, though it can be used for other purposes.
```

Comment:
This may be extremely nitpicky, but do you think it makes sense to capitalize 'integration' as well? Or perhaps lowercase 'Delivery'?